### PR TITLE
SONAR-15243 Enable json logging in k8s

### PR DESCRIPTION
**Pull request by jacek-poreda-sonarsource**

### CODE REVIEW

**Review by tobias-trabelsi-sonarsource — CHANGES_REQUESTED** [review]
Just my 2 cents here: would it make sense to provide only one flag for the json output? i can not think of a reason why somebody would want to have a structured log for the search nodes but not for the application nodes.

in any case the new parameter needs to be added to the `README.md` of the chart

**Comment by jacek-poreda-sonarsource** [comment]
Fair point, it could be one, I will add it to README as well 👍

File: charts/sonarqube-dce/values.yaml [hunk 1]
```diff
@@ -93,6 +93,10 @@ ApplicationNodes:
     # Sonarqube dockerfile creates sonarqube user as UID and GID 1000
     runAsUser: 1000
 
+  # Enable/Disable logging in JSON format
+  logging:
```
Comment (tobias-trabelsi-sonarsource) [inline charts/sonarqube-dce/values.yaml:97]: i think this should go as well right? :slightly_smiling_face:

**Review by tobias-trabelsi-sonarsource — CHANGES_REQUESTED** [review]

**Review by tobias-trabelsi-sonarsource — APPROVED** [review]
Perfect :slightly_smiling_face:

**Review by dbmeneses — APPROVED** [review]
nice :+1:

**Pull request merged**